test/rpc-integration/test-vacuous/response-vacuous-at-branch.json

```diff
@@ -9,141 +9,147 @@
                 "format": "KORE",
                 "version": 1,
                 "term": {
-                    "tag": "App",
-                    "name": "Lbl'-LT-'generatedTop'-GT-'",
-                    "sorts": [],
-                    "args": [
+                    "tag": "And",
+                    "sort": {
+                        "tag": "SortApp",
+                        "name": "SortGeneratedTopCell",
+                        "args": []
+                    },
+                    "patterns": [
```

Comment:
Unfortunately the `kore-rpc` responses for `Vacuous` results just echo the input, not splitting it into term and predicate.